### DISCUSSION TITLE
Is it possible to add pagination ?

### DISCUSSION THREAD

**Discussion by o0LINNY0o** [D4Vinci/Scrapling#1] [Q&A]
Really impressed, and is definitely faster than what I've currently been using. But my use case is that I have been scraping job sites , but the tables are only being completed on the fist page. 

I'm not a software guy but have been fumbling my way through with LLM Qwen2.5 coder with a little success. Just wondering if its worth trying to do pagination myself , and if so where would you recommend I start , or do you plan on including pagination ? 

Cheers

**Comment by D4Vinci** [comment]
Sorry, I don't understand what feature you are talking about, can you elaborate more?
Also, regarding the pagination - I'm planning to add a feature in the future to make the library extract pagination URLs when possible but other features have higher priorities right now.

**Comment by andyfcx** [comment]
I think he is talking about webpages that only loads more new data when the browser hits the bottom of the page.
I encountered this issue too.

**Reply by D4Vinci** [reply]
There are many ways to solve this from a web scraping perspective, like looking for the selector `[rel="next"]`, but from scrapling, you can use the `page_action` argument to scroll to the end of the page.
